<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VsiRefDialog</class>
 <widget class="QDialog" name="VsiRefDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Tahoma</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>VSI Reference</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../../image.qrc">
    <normaloff>:/Image/reserve.png</normaloff>:/Image/reserve.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addBtn">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../../image.qrc">
         <normaloff>:/Image/add.png</normaloff>:/Image/add.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="vsiTV">
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionEditVsiRef">
   <property name="icon">
    <iconset resource="../../../../image.qrc">
     <normaloff>:/Image/edit.png</normaloff>:/Image/edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit VSI Reference</string>
   </property>
   <property name="toolTip">
    <string>Edit VSI Reference</string>
   </property>
  </action>
  <action name="actionRemoveVsiRef">
   <property name="icon">
    <iconset resource="../../../../image.qrc">
     <normaloff>:/Image/remove.png</normaloff>:/Image/remove.png</iconset>
   </property>
   <property name="text">
    <string>Remove VSI Reference</string>
   </property>
   <property name="toolTip">
    <string>Remove VSI Reference</string>
   </property>
  </action>
  <action name="actionExport_PDF">
   <property name="icon">
    <iconset resource="../../../../image.qrc">
     <normaloff>:/Image/pdf.png</normaloff>:/Image/pdf.png</iconset>
   </property>
   <property name="text">
    <string>Export PDF</string>
   </property>
   <property name="toolTip">
    <string>Export PDF</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>addBtn</tabstop>
  <tabstop>vsiTV</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../image.qrc"/>
 </resources>
 <connections/>
</ui>
